linkml/generators/docgen/class.md.jinja2

```diff
@@ -8,6 +8,21 @@
     {%- endif -%}
 {%- endif -%}
 
+{% macro compute_range(slot) -%}
```

Comment:
this looks like logic that is already in schemaview?

Reply:
Yes, you're right, the functionality is in schemaview here: https://github.com/linkml/linkml-runtime/blob/main/linkml_runtime/utils/schemaview.py#L1471-L1486

I'm using that now